lntest/itest/lnd_routing_test.go

```diff
@@ -479,40 +457,29 @@ func testSendToRouteErrorPropagation(net *lntest.NetworkHarness, t *harnessTest)
 		Memo:  "testing",
 		Value: paymentAmt,
 	}
-	ctxt, _ = context.WithTimeout(ctxb, defaultTimeout)
-	resp, err := net.Bob.AddInvoice(ctxt, invoice)
-	if err != nil {
-		t.Fatalf("unable to add invoice: %v", err)
-	}
-
+	resp := bob.RPC.AddInvoice(invoice)
 	rHash := resp.RHash
 
-	// Using Alice as the source, pay to the 5 invoices from Bob created above.
-	ctxt, _ = context.WithTimeout(ctxb, defaultTimeout)
-	alicePayStream, err := net.Alice.SendToRoute(ctxt) // nolint:staticcheck
-	if err != nil {
-		t.Fatalf("unable to create payment stream for alice: %v", err)
-	}
+	// Using Alice as the source, pay to the 5 invoices from Bob created
+	// above.
+	alicePayStream := alice.RPC.SendToRoute()
 
 	sendReq := &lnrpc.SendToRouteRequest{
 		PaymentHash: rHash,
 		Route:       fakeRoute.Routes[0],
 	}
-
-	if err := alicePayStream.Send(sendReq); err != nil {
-		t.Fatalf("unable to send payment: %v", err)
-	}
+	err := alicePayStream.Send(sendReq)
+	require.NoError(ht, err, "unable to send payment")
 
 	// At this place we should get an rpc error with notification
 	// that edge is not found on hop(0)
 	_, err = alicePayStream.Recv()
-	if err != nil && strings.Contains(err.Error(), "edge not found") {
```

Comment:
so this is equivalent to,
```go
if err != nil {
        if !strings.Contains(err.Error(), "edge not found") {
                t.Fatalf("payment stream has been closed but fake route has consumed: %v", err)
        }
}
```

Reply:
There is a `require.ErrorContains()` function by the way that is equal to `require.Error() ... require.Contains(err.Error(), ...)`.

Reply:
Dope good to know, now switched!

Reply:
This should just be:
```
require.ErrorContains(ht, err, "payment "+
		"stream has been closed but fake route has consumed")
```

The error string lookup is now done with the incorrect one.

Reply:
Turns out the original test is a bit outdated and now fixed.